content/en/agent/basic_agent_usage/aix.md

```diff
@@ -91,7 +91,9 @@ Additionally, the following integrations can be enabled to collect further metri
 
 Enable the above integrations by copying and editing the sample configuration files provided. These are found in `/etc/datadog-agent/conf.d`. The name of the YAML configuration file should match that of the integration: `/etc/datadog-agent/conf.d/<INTEGRATION_NAME>.d/conf.yaml` enables the integration `<INTEGRATION_NAME>`, and set its configuration. Example configuration files can be found at `/etc/datadog-agent/conf.d/<INTEGRATION_NAME>.d/conf.yaml.example`
 
-Note: Some of the available metrics differ between the integrations for the Unix Agent and the integrations for Linux, Windows and MacOS. Although it is possible to monitor processes and network metrics with the Unix Agent, the Live Process Monitoring and Network Performance Monitoring capabilities aren't available.
+**Note**: Some of the available metrics differ between the integrations for the Unix Agent and the integrations for Linux, Windows and MacOS. Although it is possible to monitor processes and network metrics with the Unix Agent, the Live Process Monitoring, and Network Performance Monitoring capabilities aren't available. 
```

Comment:
I think the original punctuation in the second sentence was correct. Rather than a list of three items, the sentence reads to me like a dependent clause, followed by an independent clause that starts with a list of two items.

In other words, it is saying that while it is possible to monitor stuff with the Unix Agent, if you do so, you don't get the capabilities of Live Process Monitoring and NPM.

Reply:
nice catch yes! This was the result of going back and forth on adding Trace Agent as a third item (decided against in the end, but forgot to edit out the comma).

Reply:
```suggestion
**Note**: Some of the available metrics differ between the integrations for the Unix Agent and the integrations for Linux, Windows and MacOS. Although it is possible to monitor processes and network metrics with the Unix Agent, the Live Process Monitoring and Network Performance Monitoring capabilities aren't available. 
```